runner/src/simulate.rs

```diff
@@ -88,8 +88,19 @@ pub struct Opts {
     /// left to the scenario (requests per second, total requests, rps/node etc).
     #[arg(long, env = "SIMULATE_TARGET_REQUESTS")]
     target_request_rate: Option<usize>,
-    // #[arg(long, env = "SIMULATE_ANCHOR_WAIT_TIME")]
-    // anchor_wait_time: Option<u64>,
+
+    // Wait time after which we wosh to check the anchor correctness.
```

Comment:
typo wosh -> wish. Do you mind adding a comment about it's purpose (e.g. the scenario will not finish until this met as we wait for all streams to be anchored or whatever it is specifically)